.github/workflows/integration-test.yml

```diff
@@ -0,0 +1,94 @@
+# yaml-language-server: $schema=https://json.schemastore.org/github-workflow.json
+
+#
+# Copyright 2022 Hewlett Packard Enterprise Development LP
+# Other additional copyright holders may be indicated within.
+#
+# The entirety of this work is licensed under the Apache License,
+# Version 2.0 (the "License"); you may not use this file except
+# in compliance with the License.
+#
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+name: Integration Test
+on:
+  workflow_call:
+
+jobs:
+  test:
+    name: Run Tests
+    runs-on: ubuntu-latest
+    steps:
+
+      # Publish event file if debug is enabled
+      - name: Publish Event File
+        uses: actions/upload-artifact@v3
+        if: ${{ runner.debug }}
+        with:
+          name: integration-test-event-file
+          path: ${{ github.event_path }}
+
+      # Needed to maintain branch scope for separate docker caches
+      - name: Get Branch
+        run: echo "BRANCH=${GITHUB_REF##*/}" >> $GITHUB_ENV
+
+      - uses: actions/checkout@v2
+        with:
+          submodules: recursive
+
+      # Requireed for docker caching
+      - name: Setup Docker Buildx
+        uses: docker/setup-buildx-action@v2
+
+      # Pre-build slurm image with docker cache. This will also generate an
+      # inline cache used by the docker build in the "Integration Test" job.
+      # Docker caches are need to be image and branch scoped. The first build
+      # on new branches will need to build the image from scratch. Expect 10
+      # minutes for a full slurm build.
+      - name: Build Slurm
+        uses: docker/build-push-action@v3
+        with:
+          context: testsuite/submodules/slurm-docker-cluster
+          push: false
+          load: true
+          tags: slurm-bb:test
+          build-args: |
+            SLURM_TAG=slurm-22-05-4-1
+          cache-from: type=gha,scope=slurm-${{ env.BRANCH }}
+          cache-to: type=gha,mode=max,scope=slurm-${{ env.BRANCH }}
+
+      # Pre-build slurm image with docker cache. Expect 3 minutes for a full
+      # DWS build.
```

Comment:
When I have a release tag in DWS, you should just pull that build.

Reply:
Agreed.